app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingPrivateBrowsingViewHolder.kt

```diff
@@ -45,6 +45,7 @@ class OnboardingPrivateBrowsingViewHolder(
         }
 
         view.description_text_once.text = text
+        view.description_text_once.contentDescription = String.format(text.toString(), view.header_text.text)
         view.contentDescription = String.format(text.toString(), view.header_text.text)
```

Comment:
This should be removed. The bug was that the content description was added to the wrong view.